graphql/data/schema.js

```diff
@@ -1189,46 +1185,47 @@ const MissionType = new GraphQLObjectType({
       type: GraphQLString,
       description: 'Mission ID',
     },
-    squadName: {
+    status: {
       type: GraphQLString,
-      description: 'the name of the squad',
-    },
-    created: {
-      type: GraphQLString,
-      description: 'the timestamp from createSquadMutation',
-    },
-    started: {
-      type: GraphQLString,
-      description: 'the timestamp from first user accepting squad invite',
+      description:
+        'the current status of the current mission - pending, started, completed',
+      resolve: () => {}, //marking to calculate
     },
-    completed: {
+    squadName: {
       type: GraphQLString,
-      description: 'the timestamp from when the mission completes',
+      description: 'the name of the squad',
     },
+    //sending these both down and calculating on the front end so we can see percent move
     tabGoal: {
       type: GraphQLInt,
       description: 'the number of tabs to complete mission',
     },
-    missionType: {
-      type: GraphQLString,
-      description: 'the type of mission',
+    tabCount: {
+      type: GraphQLInt,
+      description: "the user's' number of tabs towards mission",
     },
-    acceptedSquadMembers: {
-      type: new GraphQLNonNull(new GraphQLList(GraphQLString)),
-      description: 'user ids of accepted squad members',
+    acknowledgedMissionComplete: {
+      type: GraphQLBoolean,
+      description: 'if a user has acknowledged mission complete',
+    },
+    acknowledgedMissionStarted: {
+      type: GraphQLBoolean,
+      description: 'if a user has acknowledged mission started',
     },
-    pendingSquadMembersExisting: {
+    acceptedSquadMembers: {
       type: new GraphQLNonNull(new GraphQLList(GraphQLString)),
-      description: 'user ids of pending existing users',
+      description: 'the usernames of squad members who accepted',
+      resolve: () => {},
     },
-    pendingSquadMembersEmailInvite: {
+    pendingSquadMembers: {
       type: new GraphQLNonNull(new GraphQLList(GraphQLString)),
-      description: 'email addresses of pending invited users',
+      description: 'usernames and emails of pending existing users',
+      resolve: () => {},
     },
     SquadMemberMissionStats: {
```

Comment:
I'm thinking this field should also handle accepted/pending squad members, removing the need for the above `acceptedSquadMembers` and `pendingSquadMembers` fields. The changes I'd recommend:
* Rename to `squadMembers` (reserve PascalCase for types)
* Rename `UserMissionStats` --> `SquadMember`
* Add a `status` enum to `SquadMember` (`pending`, `accepted`)
* Add `invitedEmail` to `SquadMember` -- while the squad member is pending, this would be populated with the invited user's email and `username` would be null. When the squad member accepts, `invitedEmail` would be null and the `username` would be populated. **Let's discuss:** this would share the pending user's email with the rest of the squad, which we may want to avoid.

Reply:
My thinking is that we have a mission, a mission contains total stats and squad members, and a squad member contains their own status and stats.